pkg/policy/api/l4.go

```diff
@@ -30,14 +30,10 @@ func (l4 L4Proto) IsAny() bool {
 
 // PortProtocol specifies an L4 port with an optional transport protocol
 type PortProtocol struct {
```

Comment:
To avoid complicating our lives, did we discuss the introduction of an "endPort" field, similar to what k8s network policy has? https://github.com/cilium/cilium/blob/6a70af21c3b6985096fca3f8240139b548ac4760/pkg/k8s/slim/k8s/api/networking/v1/types.go#L147-L152

Reply:
We did not.